src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -80,10 +142,21 @@
      */
     private boolean foundLadder = false;
 
-    //Stores the levels of the miners mine. This could be a map<depth,level>
+    /**
+     * Stores the levels of the miners mine. This could be a map<depth,level>
+     */
     private List<Level> levels = new ArrayList<>();
 
     /**
+     * The maximum upgrade of the building.
+     */
+    private static final int MAX_BUILDING_LEVEL = 3;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)